unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ExpressionsHUD/ExpressionsHUDController.cs

```diff
@@ -2,38 +2,56 @@
 using DCL;
 using UnityEngine;
 
+//TODO rename to EmotesHUDController
 public class ExpressionsHUDController : IHUD
```

Comment:
I think we can just rename this class in this PR?

Reply:
It will make the PR harder to review. The whole file structure, the `View`, tests and prefab will be renamed as well, but now that you have already reviewed it, I will make the change.

Reply:
Done!